### ISSUE TITLE
Response parameter 'reset' does not correspond to README

### ISSUE THREAD

**Issue by saidimu** [auth0/limitd#18]
The [README](https://github.com/auth0/limitd/blob/master/README.md) says:

```
reset: Number: A UNIX timestamp of the expected time at which the bucket will be full again
```

A response on requesting tokens from the `ip` bucket described in [limitd.yml](https://github.com/auth0/limitd/blob/master/examples/express/limitd.yml) in the `examples` folder:

```
{
  conformant: true,
  delayed: false,
  remaining: 0,
  reset: 23787223,
  limit: 10
}
```

Clearly `reset` isn't a UNIX timestamp, it would correspond to `Sat, 03 Oct 1970 07:33:43 GMT`

I'm using the latest release: [v4.1.1](https://github.com/auth0/limitd/releases/tag/v4.1.1)

**Comment by jfromaniello** [comment]
Yes, this doesn't right.

I will have a look in few hours.

 Thank you for reporting!

**Comment by dschenkelman** [comment]
@saidimu thanks for reporting. Just released version 4.1.2 with a fix for this.

Let me know if you still run into this issue as it only happened with intervals other than seconds and the sample uses seconds so it is weird that it happened while using it.

**Comment by saidimu** [comment]
@dschenkelman Works great now. Thanks!
